Misc/NEWS.d/next/Library/2023-01-12-21-22-20.gh-issue-101000.wz4Xgc.rst

```diff
@@ -0,0 +1,2 @@
+Add :func:`os.path.splitroot()`, which splits a path into a triad of
+``(drive, root, tail)``.
```

Comment:
I wonder if the NEWS entry should also mention the `pathlib` speedup; it seems significant.